openassessment/xblock/staff_area_mixin.py

```diff
@@ -272,6 +272,32 @@ def render_staff_grade_form(self, data, suffix=''):  # pylint: disable=W0613
         except PeerAssessmentInternalError:
             return self.render_error(self._(u"Error getting staff grade information."))
 
+    @XBlock.handler
+    @require_course_staff("STUDENT_GRADE")
+    def render_staff_grade_counts(self, data, suffix=''):  # pylint: disable=W0613
+        """
+        Renders a form to show the number of ungraded and checked out assessments.
+
+        Must be course staff to render this view.
+        """
+        try:
+            student_item_dict = self.get_student_item_dict()
+            course_id = student_item_dict.get('course_id')
+            item_id = student_item_dict.get('item_id')
+
+            grading_stats = staff_api.get_staff_grading_statistics(course_id, item_id)
```

Comment:
I should create a helper method here (to share with code in get_staff_path_and_context).